testapp/src/main/java/com/rakuten/tech/mobile/testapp/ui/userdata/ContactListActivity.kt

```diff
@@ -104,12 +104,10 @@ class ContactListActivity : BaseActivity() {
         if (!isFirstLaunch && settings.isContactsSaved) {
             if (settings.contactNames.isEmpty()) {
                 renderAdapter(arrayListOf())
-                checkEmpty()
             } else renderAdapter(settings.contactNames)
         } else {
             val randomList = createRandomUUIDList()
             renderAdapter(randomList)
-            settings.contactNames = randomList
```

Comment:
We shouldn't save value unless we click on Save button. So this line is redundant. From now, if the contacts are not saved, it will show a text confirmation as the screenshot given in PR's description.